### ISSUE TITLE
Consider having a spring autoconfigure

### ISSUE THREAD

**Issue by thegreystone** [opentracing-contrib/java-jfr-tracer#17]
We may want to consider having a spring autoconfigure feature for the spring folks.

**Comment by thegreystone** [comment]
@mirage22 - since you're a spring fan, I am sure this would be right up your alley. You could put a folder in the root named, for example, java-jfr-tracer-spring or java-jfr-tracer-spring-autoconfigure.

**Comment by mirage22** [comment]
well, thank you! ;) what do you think about this : java-jfr-tracer-spring-autoconfigure which is consistent with the project name java-jfr-tracer ?

**Comment by thegreystone** [comment]
No, I think we'll keep with the current naming scheme. Please sync your repo.

**Comment by thegreystone** [comment]
It is in line with how other repos have done. I.e. the repo name is java-<short-name>, the folder names correspond to <opentracing>-<short-name>.
